rotkehlchen/tasks/assets.py

```diff
@@ -113,6 +116,15 @@ def autodetect_spam_assets_in_db(user_db: DBHandler) -> None:
             if check_if_spam_token(symbol=symbol, name=name) is False:
                 continue
 
+            # check if the asset is whitelisted and skip if it is
+            with GlobalDBHandler().conn.read_ctx() as globaldb_cursor:
+                if globaldb_general_cache_exists(
+                    cursor=globaldb_cursor,
+                    key_parts=(CacheType.SPAM_ASSET_FALSE_POSITIVE,),
+                    value=identifier,
+                ) is True:
```

Comment:
nitpick: Since this is a boolean returning and boolean named function

```suggestion
                ):
```